src/Symfony/Component/Messenger/composer.json

```diff
@@ -18,7 +18,8 @@
     "require": {
         "php": ">=8.2",
         "psr/log": "^1|^2|^3",
-        "symfony/clock": "^6.4|^7.0"
+        "symfony/clock": "^6.4|^7.0",
+        "amphp/parallel": "^2.4"
```

Comment:
This must be a soft dependency (`require-dev`).

Reply:
And can you sort packages by name?

Reply:
ok!